### PR TITLE
Working CI.

### PR DESCRIPTION
**Pull request by wkoszek**

### CODE REVIEW

**Comment by paulmelnikow** [comment]
Thanks for this!

**Comment by paulmelnikow** [comment]
Thanks for this! Same note applies as #2. Will review as soon as I can, and feel free to ping me next weekend if I haven't responded by then.

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```
Comment (paulmelnikow) [inline build.sh:1]: Just a heads up that I am not keen on switching from rake to bash. Nothing against your bash of course, which looks great. In general, though, as bash scripts grow in complexity I find them very difficult to read. I've found this to be true, even when I have written them myself.

Also, on a personal note, I work infrequently enough on these projects that it's extra important to document all the project setup setups. Another reason I like rake, is that it's easy to write lots of one-liners for the random bits, and run `rake -T` for a quick reminder.

**Comment by wkoszek** [comment]
@paulmelnikow Feel free to re-do Travis build. my `.sh` I just made to "work". You can see what issue I'm hitting:

https://travis-ci.org/paulmelnikow/ObjCBSON

Have you ever tried integrating ObjCBSON in an app?

**Comment by paulmelnikow** [comment]
I'm catching up on maintenance for my iOS projects and will get this merged shortly. Sorry for the delay.

I have used ObjCBSON in a new version of ObjCMongoDB that's under development, see [this branch](https://github.com/paulmelnikow/ObjCMongoDB/tree/libmongoc). The library isn't finished, so I have used it in an application yet.

At this point there is a [Swift library](https://github.com/PureSwift/MongoDB) that seems well maintained, so I think the advantage of that project, once it's finished, is that it will have support for native Secure Transport. Of course it would be preferable for Objective-C projects as well.

**Pull request merged**